### ISSUE TITLE
Developer Tools network tab don't display request performed with bruno

### ISSUE THREAD

**Issue by Fabrice-Deshayes-aka-Xtream** [usebruno/bruno#983]
Hello.

I see that we can activate the developer tools section inside bruno (menu view -> Developer Tools).
I thought that we could see API requests performed in bruno inside the network tab of Developer Tools, but nothing appear.

Is it the expected behavior, or is it an issue?

Tests done on Bruno 1.1.0 and 1.1.1 on an up-to-date VM Manjaro Linux.

Keep the good work.

Regards.

**Comment by martinsefcik** [comment]
@Fabrice-Deshayes-aka-Xtream  Electron app has two processes: main and renderer. Developer Tools are for renderer and Bruno requests are executed in main process. So this is the reason why they are not displayed there. I think there is no option to show them there.

**Comment by Its-treason** [comment]
> Where _can_ we see raw/low-level network requests, then? That a pretty major capability I keep wanting. (Or point me to an existing Issue/Feature for that)

What raw/low-level request data are you missing?

**Comment by dougbreaux** [comment]
When I click to send a request, I'd like to see the entire raw HTTP request, headers and all, somewhere.

Ideally, I'd like to see the entire raw HTTP response as well, but at least there are tabs for the parts.

That said, though, it looks like Timeline has what I want. (I'd propose a different name, but that's not crucial)

**Comment by martinsefcik** [comment]
@dougbreaux Timeline tab does not show all request headers but only these which were set on axios level.

**Comment by dougbreaux** [comment]
In which case, I'd still like a "raw" request/response view, then. 😄 I think all the other API tools I've used have that capability. If this is not the Issue for that, and there isn't another one, I can open one.

**Comment by thekalinga** [comment]
@martinsefcik Is this a security restriction of electron (or) Can it be customised?

**Comment by codedog** [comment]
It's been a while since this ticket was updated. I am also missing this functionality, i.e. being able to see the raw HTTP request with all the headers.

**Comment by jims-code** [comment]
Is there any external tool that can be used as workaround? E.g. Mitmproxy or Wireshark? API Testing would be much easier if network traffic would be visible in Bruno, so I would be very happy if this feature would be available.

**Comment by andrei-korshikov** [comment]
> Is there any external tool that can be used as workaround? E.g. Mitmproxy

Exactly. Look at [mitmproxy](https://mitmproxy.org/) or [HTTP Toolkit](https://httptoolkit.com/).

> or Wireshark?

Of course, but be aware of #2037.

**Comment by anusree-bruno** [comment]
Since this issue was created, we've improved the timeline view of requests to log the network calls between them. 

https://github.com/user-attachments/assets/bad4e9d6-82e6-4b56-a4f4-a8e2da87b8d8

Additionally, we're working on a built-in debug console in Bruno—[PR #4922](https://github.com/usebruno/bruno/pull/4922)—which will introduce a Network tab where you can view all logs.

**Comment by naman-bruno** [comment]
Hi @Fabrice-Deshayes-aka-Xtream,
We've introduced a Network tab inside DevTools in version 2.8.0, which displays all requests made through Bruno along with detailed information. This should hopefully resolve the issue you were facing.
Please update to the latest version and give it a try!

<img width="1344" height="356" alt="Image" src="https://github.com/user-attachments/assets/263eae10-3e65-403a-9577-74d1401087ba" />
